js/ui/validation_message.js

```diff
@@ -122,30 +122,48 @@ const ValidationMessage = Overlay.inherit({
         this.option({ maxWidth });
     },
 
+    _getPositionsArray: function(positionRequest, positionSide) {
```

Comment:
let's rename `positionRequest` to `position` everywhere

Reply:
mmkey

Reply:
renamed to `positionSide` becase VM already has `position` option